src/poolMetas.ts

```diff
@@ -29,6 +32,9 @@ export let poolMetas: PoolMeta[] = [
     nftFeed: '0xab351d3e54c975bfa0c2edafb6fab03f94762111',
     assessor: '0x1aba642c1aac9f8da36f7df73eda4ca73e054084',
     seniorTranche: '0x4c1bfb4e3ecbd6200358038e3f560ab6dee9bcb6',
+    juniorTranche: null,
```

Comment:
I proposed to Jenny not to set these here because we're replacing this with the on chain registry soon anyway and this is for v2 pools where this won't be used anyway.